src/checkbox/README.md

```diff
@@ -2,7 +2,33 @@
 
 The `vscode-checkbox` is a web component implementation of a [checkbox element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox).
 
-## Attributes
+![Checkbox hero](/docs/assets/checkbox-hero.png)
+
+## Usage
+
+| ❌ Don't                                               | ✅ Do                                                             |
+| ------------------------------------------------------ | ----------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png) | ![Image placeholder](/docs/assets/img-placeholder.png)            |
+| Don't group unrelated checkboxes together.             | Introduce a group of related checkboxes with a descriptive label. |
+
+| ❌ Don't                                               | ✅ Do                                                                           |
+| ------------------------------------------------------ | ------------------------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png) | ![Image placeholder](/docs/assets/img-placeholder.png)                          |
+| Avoid using horizontal checkbox group layouts.         | Arrange checkbox groups vertically to make it easy to scan and compare options. |
+
+| ❌ Don't                                                                                                        | ✅ Do                                                  |
+| --------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| ![Image placeholder](/docs/assets/img-placeholder.png)                                                          | ![Image placeholder](/docs/assets/img-placeholder.png) |
+| Don't use checkboxes for options with more than two clear states. Consider using a radio button for such cases. | Use checkbox titles that imply opposite states.        |
```

Comment:
Could you explain this guidance a bit more? I'm a bit confused because radio/radio groups only have the option to select one radio item and state.

Although I think there might be confusion on my part about what "state" means.